### PR TITLE
feat: 🎸 Update react-vtkjs-viewport usage to use requestPool

### PR DESCRIPTION
**Pull request by JamesAPetts**
RE: #1957  https://github.com/OHIF/react-vtkjs-viewport/pull/103

Switched to using the `cornerstone-tools` `requestPoolManager`, so that we don't send 1000 requests at the same time to the PACS on `imageDataObject.loadImage()`.

This is a breaking change, as we the `imageDataObject` no-longer has `insertPixelDataPromises`, instead the consumer may subscribe to an `imageDataObject`'s o`nPixelDataInserted` and/or `onAllPixelDataInserted `hooks. All the relavent examples have been updated to reflect this. As we are still pre-1.0, I have avoided a major version bump.

### CODE REVIEW

**Comment by dannyrb** [comment]
@JamesAPetts, do you need a review?

**Comment by JamesAPetts** [comment]
@dannyrb Sure, bear in mind this is just an update to the new API introduced in https://github.com/OHIF/react-vtkjs-viewport/pull/103 to function in OHIF, but feel free to take a look.

File: extensions/vtk/src/OHIFVTKViewport.js [hunk 1]
```diff
@@ -358,36 +360,38 @@ class OHIFVTKViewport extends Component {
   }
 
   loadProgressively(imageDataObject) {
+    debugger;
```
Comment (swederik) [inline extensions/vtk/src/OHIFVTKViewport.js:363]: bad @JamesAPetts

**Comment by swederik** [comment]
LGTM now that the debugger is gone

**Pull request merged**